### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by kkraune**

### CODE REVIEW

File: en/tutorials/news-7-recommendation-with-parent-child.md [hunk 1]
```diff
@@ -233,9 +233,8 @@ is multiplied by the `1.0` in this document. So in this case, this would
 effectively work as a lookup.
 
 <p class="alert alert-success"> 
-Much more complex operations are available, please refer to [the tensor 
-user guide](../tensor-user-guide.html#ranking-with-tensors) for more
-information.
+Much more complex operations are available, please refer to <a href="../tensor-user-guide.html#ranking-with-tensors">
+the tensor user guide</a> for more information.
```
Comment (lesters) [inline en/tutorials/news-7-recommendation-with-parent-child.md:237]: Is this change necessary? Can't see much of a difference, except the use of `<a` which is inconsistent with the rest of the document?

**Review by lesters — APPROVED** [review]

File: en/tutorials/news-7-recommendation-with-parent-child.md (hunk 1 above)
Comment (kkraune) [reply en/tutorials/news-7-recommendation-with-parent-child.md:237]: it is inside a &lt;p&gt; so markdown will not parse it to a link, as observed at https://docs.vespa.ai/en/tutorials/news-7-recommendation-with-parent-child.html

Comment (lesters) [reply en/tutorials/news-7-recommendation-with-parent-child.md:237]: 👍
